modelEvaluation/compare.py

```diff
@@ -158,9 +174,9 @@ def performance_distribution(models):
     pass #TODO  write
 #%%
 if __name__=='__main__':
-    year=int(input('YEAR TO PREDICT: '))
-    nested=eval(input('NESTED MODEL COMPARISON? (True/False) '))
-    all_models=eval(input('COMPARE ALL MODELS? (True/False) '))
+    year=int(input('YEAR TO PREDICT: ')) if not hasattr(args, 'year') else args.year
+    nested=eval(input('NESTED MODEL COMPARISON? (True/False) ')) if not hasattr(args, 'nested') else args.nested
```

Comment:
This is probably wrong! Should be: if not args.nested, because we are parsing booleans with default values